samples/excel/85-preview-apis/range-merged-areas.yaml

```diff
@@ -0,0 +1,120 @@
+order: 7
+id: excel-merged-areas
+name: Merged areas
+description: This sample shows how to create and then find the merged areas in a worksheet.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.13'
+script:
+    content: |
+      $("#setup").click(() => tryCatch(setup));
+      $("#get-merged-areas").click(() => tryCatch(getMergedAreas));
+
+      async function getMergedAreas() {
+        await Excel.run(async (context) => {
+          // Retrieve the worksheet and the table in that worksheet.
+          const sheet = context.workbook.worksheets.getActiveWorksheet();
+          const table = sheet.getRange("B2:E6");
+          
+          // Retrieve the merged areas within the table and load their details. 
+          let mergedAreas = table.getMergedAreasOrNullObject();
+          mergedAreas.load("address");
+          mergedAreas.load("cellCount");
+
+          // For visibility, highlight the merged area located at cell B2. 
+          const range = sheet.getRange("B2");
+          range.select();
+          await context.sync();
+
+          // Print out the details of the `mergedAreas` range object. 
+          console.log(`Address of the merged area: ${mergedAreas.address}`);
+          console.log(`Number of cells in the merged area: ${mergedAreas.cellCount}`);
+
+          await context.sync();
+        });
+      }
+
+      async function setup() {
+        await Excel.run(async (context) => {
+          context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+          const sheet = context.workbook.worksheets.add("Sample");
+
+          // Create a table. 
+          const data = [
+            ["Product chart", "", "", ""],
+            ["Product", "Qty", "Unit Price", "Total Price"],
+            ["Almonds", 2, 7.5, "=C3 * D3"],
+            ["Coffee", 1, 34.5, "=C4 * D4"],
+            ["Chocolate", 5, 9.56, "=C5 * D5"]
+          ];
+
+          const range = sheet.getRange("B2:E6");
+          range.values = data;
+          range.format.autofitColumns();
+
+          // Add a header row to the table.
+          const header = range.getRow(1);
+          header.format.fill.color = "#4472C4";
+          header.format.font.color = "white";
+
+          // Create a merged area above the header row of the table. 
+          const chartTitle = range.getRow(0);
+          chartTitle.merge(true);
+          chartTitle.format.horizontalAlignment = "Center";
```

Comment:
If this bit was pulled into a function and given a button, the sample could be generalized to a "Merged ranges" sample.